From felipe.balbi@nokia.com  Fri Jun  6 15:12:15 2008
From: Felipe Balbi <felipe.balbi@nokia.com>
Date: Thu, 22 May 2008 02:45:11 +0300
Subject: USB: COMPOSITE: Support 16 interfaces by default
To: linux-usb@vger.kernel.org
Cc: David Brownell <dbrownell@sourceforge.users.net>, Felipe Balbi <felipe.balbi@nokia.com>
Message-ID: <1211413513-1320-2-git-send-email-felipe.balbi@nokia.com>


8 interfaces is not enough if we try to use
3 instances of obex function driver.

Signed-off-by: Felipe Balbi <felipe.balbi@nokia.com>
Acked-by: David Brownell <dbrownell@users.sourceforge.net>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 include/linux/usb/composite.h |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/include/linux/usb/composite.h
+++ b/include/linux/usb/composite.h
@@ -147,7 +147,7 @@ ep_choose(struct usb_gadget *g, struct u
 	return fs;
 }
 
-#define	MAX_CONFIG_INTERFACES		8	/* arbitrary; max 255 */
+#define	MAX_CONFIG_INTERFACES		16	/* arbitrary; max 255 */
 
 /**
  * struct usb_configuration - represents one gadget configuration
